qutebrowser/mainwindow/tabwidget.py

```diff
@@ -172,6 +173,12 @@ def get_tab_fields(self, idx):
         fields['perc_raw'] = tab.progress()
         fields['backend'] = objects.backend.name
         fields['private'] = ' [Private Mode] ' if tab.private else ''
+        try:
+            fields['audio'] = '[M] ' if tab.is_muted() else (
+                '[A] ' if tab.is_recently_audible() else '')
+        except (NotImplementedError, browsertab.WebTabError):
+            # one of the functions was not implemented or had an error, abort
```

Comment:
I don't think that comment is helpful - but "Only implemented with QtWebEngine" would be :wink: